ci/buildwheel.sh

```
@@ -0,0 +1,64 @@
+  if [ x"${BUILD_TYPE}" == x"release" ]; then
+    upload_lib
+  elif [ x"${BUILD_TYPE}" == x"local" ]; then
+    echo " copy into /workspace dir, which is a docker volume mount"
+    rm -rf  /workspace/{build,dist}
+    cp -avr /app/build /app/dist /workspace/
+  else
+    echo "Discard builded library!"
+  fi
+}
+
+main $@
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)